### ISSUE TITLE
Getting started page uses outdated psci examples

### ISSUE THREAD

**Issue by Unisay** [purescript/purescript.github.io#117]
[Getting started](http://www.purescript.org/learn/getting-started/) page uses psci examples with `let` keyword which are no longer required as of version 0.11.0:

```
> let ns = range 0 999
(line 1, column 18):
unexpected end of input
expecting ., {, ::, operator, "where", indentation at column 5 or "in"
Declarations in PSCi no longer require "let", as of version 0.11.0
```

Another example uses outdated list representation:
```
> multiples
Cons 0 (Cons 3 (Cons 5 (Cons 6 (Cons ...
```
while actual psci shows it differently:

```
> multiples
(0 : 3 : 5 : 6 : 9 : 10 : 12 : 15 : 18 : 20 :
```

**Comment by Unisay** [comment]
May I ask for a comment on why is the issue closed? It's not clear if it is fixed or invalid or anything else. If the issue is invalid (bad description, wrong repo or duplicate or something like this I'd like to know it in order not to repeat the mistake)

**Comment by soupi** [comment]
The issue was closed because the commit that fixed the issues you reported #118 has been merged :) 
you can see the reference for it above the closing comment in this thread.

**Comment by Unisay** [comment]
@soupi Thank you, I was using mobile application (OctoDroid) for github, that didn't show the referenced commit. Now (from the browser) its clear.
